app/viewtils/EvidenceFileUploadsHelper.scala

```diff
@@ -60,4 +65,31 @@ class EvidenceFileUploadsHelper @Inject()(govukInsetText: GovukInsetText,
     }
 
   }
+
+  def getInsetTextMessage(journeyId: String)(implicit messages: Messages): Future[Option[String]] = {
+      uploadJourneyRepository.getUploadsForJourney(Some(journeyId)).map(_.fold[Option[String]](None)(
+        uploads => {
+          val uploadedFiles: Seq[(UploadJourney, Int)] = uploads.zipWithIndex
+          val mappedDuplicates: Map[Option[String], Seq[(UploadJourney, Int)]] = uploadedFiles.groupBy(_._1.uploadDetails.map(_.checksum))
+
+          if (!mappedDuplicates.exists(_._2.size > 1)) {
+            None
+          } else if (mappedDuplicates.count(_._2.size > 1) > 1) {
+           Some(messages("otherReason.uploadList.multipleDuplicateInsetText", mappedDuplicates.map(_._2.size).sum))
+          } else {
+            val duplicates: Seq[(UploadJourney, Int)] = mappedDuplicates.filter(_._2.size > 1).head._2
```

Comment:
Lines 81 to 88, written below to find the inset text is same as the lines 55 to 62 in the method 'duplicateDocumentInset'. Can these be refactored.